modules/navier_stokes/src/fvbcs/INSFVSymmetryPressureBC.C

```diff
@@ -11,14 +11,15 @@
 #include "InputParameters.h"
 
 registerMooseObject("NavierStokesApp", INSFVSymmetryPressureBC);
+registerMooseObject("NavierStokesApp", INSFVSymmetryScalarBC);
 
 InputParameters
 INSFVSymmetryPressureBC::validParams()
 {
   auto params = INSFVSymmetryBC::validParams();
-  params.addClassDescription("Though applied to the pressure, this object ensures that the "
-                             "velocity perpendicular to a symmetry bounadry is zero by setting the "
-                             "mass flow rate across the symmetry boundary to zero.");
+  params.addClassDescription("Though not applied to velocity, this object ensures that the "
+                             "velocity perpendicular to a symmetry boundary is zero by setting "
+                             "the mass/scalar flow rate across the symmetry boundary to zero.");
```

Comment:
I'm still not sure I love the description. Setting the scalar flux to zero at the boundary just requires that the product of rho, velocity, and the scalar concentration is zero. This could be accomplished by either the velocity *or* the scalar concentration being zero, so this object applied to a scalar flow doesn't guarantee that the velocity will be zero. Only application of this object to the pressure will ensure zero velocity perpendicular to the symmetry boundary since in the product of rho and velocity, only velocity can be zero.

Reply:
Ok I ll fix that